src/core/agent/event.types.ts

```diff
@@ -54,6 +55,13 @@ interface ConnectionStateChangedEvent extends BaseEventEmitter {
   };
 }
 
+interface OobiResolvedEvent extends BaseEventEmitter {
```

Comment:
This event is for a new pending connection, not OOBI resolved... We don't need a new interface here. Can we not add 2 event callbacks to the same `EventTypes.ConnectionStateChangedEvent`?